### PR TITLE
[IMP] Add action on partner tree to add several partners to an event

### PR DESCRIPTION
**Pull request by antespi**
This PR improve partner_event addon, adding:
- Action in partner tree view 'More' button, to register several partners to an event

It also include, README fixes suggested by @pedrobaeza in https://github.com/OCA/event/pull/5

### CODE REVIEW

File: partner_event/wizard/res_partner_register_event.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+            'email': partner.email,
+            'phone': partner.phone,
+            'date_open': fields.Datetime.now(),
+        }
+
+    @api.multi
+    def button_register(self):
+        registration_obj = self.env['event.registration']
+        partner_obj = self.env['res.partner']
+        print self.env.context.get('active_ids', [])
```
Comment (pedrobaeza) [inline partner_event/wizard/res_partner_register_event.py]: Remove this line

**Comment by pedrobaeza** [comment]
Little remark, but overall :+1:

You can make a test exercising the mass-registration anyway.

File: partner_event/wizard/res_partner_register_event.py (hunk 1 above)
Comment (antespi) [inline partner_event/wizard/res_partner_register_event.py]: Done, thanks.

**Comment by JavierIniesta** [comment]
Tested on local instance. :+1:

**Comment by janetrp** [comment]
tested, it works! :+1:

**Pull request merged**